store/driver/txn/txn_driver.go

```diff
@@ -126,6 +126,8 @@ func (txn *tikvTxn) SetOption(opt int, val interface{}) {
 			txn:     txn.KVTxn,
 			binInfo: val.(*binloginfo.BinlogInfo), // val cannot be other type.
 		})
+	case tikvstore.SyncLog:
+		txn.SetSyncLog()
```

Comment:
Would there need a situation that we want to disable `synclog` in the future?

Reply:
S

> Would there need a situation that we want to disable `synclog` in the future?

It seems we do not need this situation. However, I think renaming the function name to `EnableSyncLog` instead of `SetSyncLog` would be better?

Reply:
em. How about `EnableForceSynclog`? should be more accurate.